test/Templates.Test/Helpers/AddFirewallExclusion.cs

```diff
@@ -9,16 +9,11 @@ internal class AddFirewallExclusion : IDisposable
         private bool _disposedValue = false;
         private readonly string _exclusionPath;
 
-        public static IDisposable Create(string exclusionPath)
-        {
-            return new AddFirewallExclusion(exclusionPath);
-        }
-
-        private AddFirewallExclusion(string exclusionPath)
+        public AddFirewallExclusion(string exclusionPath)
```

Comment:
With this change to the API, there's a risk that `AddFirewallExclusion` could behave differently than a developer might expect. Since the method is called `Add...`, it suggests that you can call it multiple times and add multiple firewall exclusions. But if you do that, then `Dispose` won't do what you'd expect (it will only remove the last one you added). I'd suggest that `exclusionPath` should only be passed to the constructor, and not modifyable afterwards. In fact you probably only need a constructor, and don't need a separate `AddFirewallExclusion` call.